arangod/Cluster/Utils/PlanCollectionEntryReplication2.cpp

```diff
@@ -66,17 +66,17 @@ PlanShardToServerMapping PlanCollectionEntryReplication2::getShardMapping()
 
 [[nodiscard]] replication2::agency::LogTarget
 PlanCollectionEntryReplication2::getReplicatedLogForTarget(
-    ShardID const& shardId, ResponsibleServerList const& serverIds,
-    std::string_view databaseName) const {
+    replication2::LogId const& logId, ResponsibleServerList const& serverIds,
+    std::string_view databaseName, ShardID const& shardId) const {
   using namespace replication2::replicated_state;
 
   replication2::agency::LogTarget spec;
 
-  spec.id = LogicalCollection::shardIdToStateId(shardId);
```

Comment:
Is the shardIdToStateId function now obsolete?

Maybe we can already remove it now.

Reply:
It is still needed unfortunately, for `ClusterInfo` methods which rely on the old mapping between `shardId` and `logId`. After we manage to move that to collection groups, we'll safely remove it.